swagger/swagger.yaml

```diff
@@ -0,0 +1,2624 @@
+swagger: '2.0'
```

Comment:
Does this get auto generated whenever the chain starts with proto? Or do devs have to run `proto-swagger-gen`?

Reply:
right now, if we change the proto files we need to run `proto-swagger-gen` to update the `swagger.yaml`